Mathlib/Data/Nat/Choose/Mul.lean

```diff
@@ -0,0 +1,48 @@
+/-
+Copyright (c) 2024 Antoine Chambert-Loir & María-Inés de Frutos—Fernández. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Antoine Chambert-Loir, María-Inés de Frutos—Fernández
+-/
+
+import Mathlib.Data.Nat.Choose.Basic
+import Mathlib.Tactic.Ring.RingNF
+
+/-! # Two lemmas on choose
+
+The proofs of these lemmas use the `ring` tactic
+and can't be given in `Mathlib.Data.Nat.Choose.Basic`
+
+-/
+
+namespace Nat
+
+theorem choose_mul_add (m) {n : ℕ} (hn : n ≠ 0) :
```

Comment:
I would suggest putting `hn` first since `m` will usually be inferrable from context

Reply:
I made the `m` argument implicit. That works, and I preferred having the arguments in the same order as they appear in the formula.